### DISCUSSION TITLE
Would you like to look over the new `ki` API before it's published?

### DISCUSSION THREAD

**Discussion by mitchellwrosen** [evanrelf/drama#11] [General]
Hello! I happened to notice `drama` is the _one_ library on Hackage that depends on `ki` :)

I'm not sure if you are planning on using `ki` long-term, or if you are just experimenting with different flavors of concurrency primitives - but I'd figure I'd at lease point you at the release candidate docs.

https://hackage.haskell.org/package/ki-0.3.0/candidate/docs/Ki.html

Let me know if it's working out! Thanks :)

**Comment by evanrelf** [comment]
Hi @mitchellwrosen 🙂 Thanks for making `ki`!

I've really enjoyed using the library, and have no plans at the moment to move away from it. It suits the actor model style really well.

As far as my library migrating to `ki` 0.3.0 is concerned, it looks like it should be easy:

- `Ki.Implicit` goes away, but I wasn't using it
- I'm passing `IO ()` actions (which may return at some point) to `fork_`, so I'll need to switch to `fork` and ignore the `Thread` it returns
- `wait scope` becomes `atomically $ wait scope`, that's fine

**Reply by mitchellwrosen** [reply]
Great, thanks!
